### ISSUE TITLE
[Topic Entry] Python math-module math.acosh()

### ISSUE THREAD

**Issue by codecademy-docs** [Codecademy/docs#5077]
### Reason/inspiration (optional)

We would like a new term entry on the `math.acosh()` method of the [`math-module`](https://www.codecademy.com/resources/docs/python/math-module) concept in Python. The entry should go in a new file under `docs/content/python/concepts/math-module/terms/math-acosh/math-acosh.md`

The entry should include the following:

- A description of the term
- A ##Syntax section that includes the syntax of the method and its details
- An ##Example section that includes a ```py ``` codeblock and ```shell ``` that shows the example output.
- A ##Codebyte Example to show the working of the method

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Topic Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

**Comment by mamtawardhani** [comment]
@FabioNH86 you are assigned to this one, you have already raised a PR for this.

**Comment by mamtawardhani** [comment]
> Hey @mamtawardhani, though we're already considering the author as the assignee since he has raised a PR for it, it seems that we haven't yet assigned this issue to him manually.
> 
> So @FabioNH86, please comment on this issue with something like 'I would like to work on this issue' so that you can be assigned.

Yup, he had commented on a different issue but raised a different PR, so even I've asked him to comment on the issue. @FabioNH86 please do the needful 😃

**Comment by FabioNH86** [comment]
Hello! I would like to work on this issue

**Comment by Sriparno08** [comment]
Hey @FabioNH86, you're assigned!

**Comment by FabioNH86** [comment]
What do I have to do now? I noticed that the other PR was successfully merged 😵‍💫 Sorry

**Comment by mamtawardhani** [comment]
Hey @FabioNH86, your pr is in review and will be merged soon. No action needed from your side 😃
